src/batou_ext/redis.py

```diff
@@ -28,3 +33,13 @@ def configure(self):
         self += batou.lib.file.File(
             self.password_file, content=self.password, sensitive_data=True
         )
+        if self.cleanup:
+            self += batou.lib.file.File(
+                "rediscleanup.sh",
+                source=self.resource("rediscleanup.sh"),
+                mode=0o755,
+            )
+            self += batou_ext.run.Run("rediscleanups.sh", file=self._)
```

Comment:
Eventhough this is just kind of a marker inside breadcrumbs, this typo puzzles me